src/main/groovy/pl/allegro/tech/build/axion/release/domain/VersionConfig.groovy

```diff
@@ -96,11 +96,15 @@ class VersionConfig {
     VersionWithPosition getRawVersion() {
         if (rawVersion == null) {
             ensureVersionServiceExists()
-            rawVersion = versionService.currentVersion(this, VersionReadOptions.fromProject(project))
+            rawVersion = versionService.currentVersion(this, getVersionReadOptions())
         }
         return rawVersion
     }
 
+    VersionReadOptions getVersionReadOptions() {
```

Comment:
Okay, after giving it a bit of thought - if we carry on with this PR (see my comment for the issue), the `VersionConfig$getVersionReadOptions` should be removed/kept private and `Releaser` should remain unchanged. I don't want `VersionConfig` to expose anything more than absolute minimum (even if it means crossing the DRY).